qpdf.yaml

```diff
@@ -0,0 +1,71 @@
+# Generated from https://git.alpinelinux.org/aports/plain/community/qpdf/APKBUILD
+package:
+    name: qpdf
+    version: 11.8.0
+    epoch: 0
+    description: Command-line tools and library for transforming PDF files
+    copyright:
+        - license: Apache-2.0
+environment:
+    contents:
+        packages:
+            - autoconf
+            - automake
+            - build-base
+            - busybox
+            - ca-certificates-bundle
+            - cmake
+            - gnutls-dev
+            - libjpeg-turbo-dev
+            - openssl-dev
+            - samurai
+            - zlib-dev
+pipeline:
+    - uses: fetch
+      with:
+        expected-sha256: d9321f5fbc50251803630a5604ddc5ed9a4d93bc023d9a7436a302e7c9741259
+        uri: https://github.com/qpdf/qpdf/releases/download/v${{package.version}}/qpdf-${{package.version}}.tar.gz
+    - name: Build
+      runs: |
+        CXXFLAGS="$CXXFLAGS" \
```

Comment:
Are these 2 lines needed?

Reply:
nope I removed one additional parameter from them so its ok to remove them